src/plugin/fieldViews/SelectFieldView.ts

```diff
@@ -0,0 +1,32 @@
+import type { Node } from "prosemirror-model";
+import type { EditorView } from "prosemirror-view";
+import { CustomFieldView } from "./CustomFieldView";
+import { FieldType } from "./FieldView";
+
+/**
+ * A FieldView representing a
+ * node that contains arbitrary fields that are updated atomically.
+ *
+ * Instead of rendering into a DOM node that's mounted by the consumer, this FieldView
+ * instead provides a `subscribe` method that allows consuming code to listen for
+ * state changes. In this way, consuming code can manage state and UI changes itself,
+ * perhaps in its own renderer format.
+ */
+export class SelectFieldView<Fields = unknown> extends CustomFieldView<Fields> {
```

Comment:
Is this still used? I can't see a reference elsewhere in the PR.

Reply:
Whoops - I must have left that in during early experimentation and didn't realise I committed it.

Will get rid of it now.